consensus/polybft/polybft_config.go

```diff
@@ -75,21 +77,40 @@ func (p *PolyBFTConfig) IsBridgeEnabled() bool {
 
 // Validator represents public information about validator accounts which are the part of genesis
 type Validator struct {
-	Address types.Address
-	BlsKey  string
-	Balance *big.Int
-	NodeID  string
+	Address       types.Address
+	BlsPrivateKey *bls.PrivateKey
+	BlsKey        string
+	BlsSignature  string
+	Balance       *big.Int
+	NodeID        string
 }
 
 type validatorRaw struct {
-	Address types.Address `json:"address"`
-	BlsKey  string        `json:"blsKey"`
-	Balance *string       `json:"balance"`
-	NodeID  string        `json:"nodeId"`
+	Address      types.Address `json:"address"`
```

Comment:
Why do we need this struct? Can't we just have json annotations on the `Validator` struct, and use it to marshal and unmarshal? We can ignore some of the fields if we don't need to marshal them.

Reply:
This is done by @Stefan-Ethernal because of `big.Int` serialization problem. We can change that in some other PR if we found better solution